lambdas/send_email/src/send_email_queries.py

```
@@ -1,22 +1,16 @@
 from hyp3_db.hyp3_models import User, OneTimeAction, Subscription
 
 
-def get_user_by_email(db, email):
-    return db.session.query(User) \
-        .filter_by(email=email) \
-        .first()
-
-
 def get_user_by_id(db, user_id):
     return db.session.query(User) \
         .filter_by(id=user_id) \
-        .first()
+        .one()
```

`.one()` throws an exception if the query returns no results (really an amount other than 1 result). As the calling code still checks for `None` being returned, I doubt this is the behavior you want.

`.first()` on the other hand always returns either 1 item or None if there were not result.

---

Got an error later in the code from the sub being none.

---

Might as well fail early.

---

This returns a User not a Sub, so if the Sub was None then this was not the cause of the error.